apps/server/src/infra/schulconnex-client/response/sanis-erreichbarkeiten-response.ts

```diff
@@ -0,0 +1,11 @@
+import { IsOptional, IsString } from 'class-validator';
+
+export class SanisErreichbarkeitenResponse {
+	@IsOptional()
+	@IsString()
+	typ?: string;
```

Comment:
This could be an enum, since it is a "code"-type in schulconnex found under 11.19 Erreichbarkeitstyp